src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -7,61 +7,234 @@ import {
+  },
+  {
+    lovelaceCardConfig: {
+      type: "markdown",
+      content:
+        "The **Markdown** card allows you to write any text. You can style it **bold**, *italicized*, ~strikethrough~ etc. You can do images, links, and more. For more information see the [Markdown Cheatsheet](https://commonmark.org/help).",
+    },
+    name: "Markdown",
+    noEntity: true,
+  },
+  {
+    lovelaceCardConfig: { type: "media-control" },
+    includeDomains: ["media_player"],
+    name: "Media Control",
+  },
+  {
+    lovelaceCardConfig: {
+      type: "picture",
+      image:
+        "https://www.home-assistant.io/images/merchandise/shirt-frontpage.png",
+    },
+    name: "Picture",
+    noEntity: true,
+  },
+  {
+    lovelaceCardConfig: {
+      type: "picture-entity",
+      image:
+        "https://www.home-assistant.io/images/merchandise/shirt-frontpage.png",
+    },
+    name: "Picture Entity",
+  },
+  {
+    lovelaceCardConfig: {
+      type: "picture-glance",
+      image:
+        "https://www.home-assistant.io/images/merchandise/shirt-frontpage.png",
+    },
+    numberOfEntities: 2,
+    name: "Picture Glance",
+  },
+  {
+    lovelaceCardConfig: {
+      type: "plant-status",
+    },
+    includeDomains: ["plant"],
+    name: "Plant Status",
+    description: "A card for all the lovely botanists out there.",
+  },
+  {
+    lovelaceCardConfig: { type: "sensor", graph: "line" },
+    includeDomains: ["sensor"],
+    name: "Sensor",
+  },
+  {
+    lovelaceCardConfig: { type: "shopping-list" },
+    name: "Shopping List",
+    description:
+      "The Shopping List Card allows you to add, edit, check-off, and clear items from your shopping list.",
+    noPreview: true,
+  },
+  {
+    lovelaceCardConfig: { type: "thermostat" },
+    includeDomains: ["climate"],
+    name: "Thermostat",
+  },
+  {
+    lovelaceCardConfig: { type: "weather-forecast" },
+    includeDomains: ["weather"],
+    name: "Weather Forecast",
+  },
+  {
+    lovelaceCardConfig: { type: "vertical-stack" },
+    name: "Vertical Stack",
+    description:
+      "Vertical stack allows you to group multiple cards so they always sit in the same column.",
+    noPreview: true,
+  },
 ];
 
 @customElement("hui-card-picker")
 export class HuiCardPicker extends LitElement {
   public hass?: HomeAssistant;
-
+  public lovelace?: LovelaceConfig;
+  public entities?: string[];
   public cardPicked?: (cardConf: LovelaceCardConfig) => void;
 
   protected render(): TemplateResult {
     return html`
+      <h2>Main</h2>
       <div class="cards-container">
-        ${cards.map((card: string) => {
-          return html`
-            <mwc-button @click="${this._cardPicked}" .type="${card}">
-              ${this.hass!.localize(
-                `ui.panel.lovelace.editor.card.${card}.name`
+        ${cardConfigs
+          .filter((cardConfig) => !cardConfig.noPreview)
+          .map((cardConfig: CardPickerConfig) => {
+            return html`
+              ${until(
```

Comment:
Using Async and until so we can get the stubs from each card. Easier to maintain defaults within each card